client/js/libs/handlebars/tojson.js

```diff
@@ -1,7 +1,3 @@
-"use strict";
-
-Handlebars.registerHelper(
-	"toJSON", function(context) {
-		return JSON.stringify(context);
-	}
-);
+module.exports = function(context) {
+	return window.JSON.stringify(context);
+};
```

Comment:
This file needs to be renamed to `toJSON.js`, otherwise the handlebars loader will be unable to resolve it on case-sensitive file systems.